src/Microsoft.Extensions.EnvironmentAbstractions/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "Abstractions for making code that uses file system and environment testable.",
```

Comment:
Renamed the project make all of it internal, unfortunately this required putting InternalsVisibleTo in couple more place.